src/proc.ts

```diff
@@ -76,6 +76,8 @@ export interface BuildCommand {
 
 export interface EnvironmentVariables { [key: string]: string }
 
+export type EnvironmentMapType = EnvironmentVariables | Map<string, string> | NodeJS.ProcessEnv;
```

Comment:
the name is confusing. Could you change it to something like GeneralEnvironmentType?

Reply:
done